sql/core/src/test/resources/sql-tests/inputs/auto_join0.sql

```diff
@@ -0,0 +1,9 @@
+select sum(hash(a.k1,a.v1,a.k2, a.v2))
```

Comment:
we don't guarantee hash consistency across versions so I'm not sure if it is a good idea to use hash here.